torchci/scripts/check_alerts.py

```diff
@@ -221,6 +222,38 @@ def generate_failed_job_issue(failed_jobs: List[JobStatus]) -> Any:
     return issue
 
 
+def gen_update_comment(original_body: str, jobs: List[JobStatus]) -> str:
+    """
+    Returns empty string if nothing signficant changed. Otherwise returns a
+    short string meant for updating the issue.
+    """
+    regex_pattern = r'^- \[(.*)\]\(.*\) failed consecutively starting with commit \[.*\]\(.*\)$'
```

Comment:
Q: Where this string is generated? Perhaps it can be moved to top of the file as `RE_FAILED_CONSEQUENTIVELY`?

Reply:
its generated in the function right about this one